librz/core/cmd_descs/cmd_type.yaml

```diff
@@ -162,6 +197,35 @@ commands:
         args:
           - name: file
             type: RZ_CMD_ARG_TYPE_FILE
+  - name: tp
+    summary: Print formatted type casted to the address
+    subcommands:
+      - name: tp
+        cname: type_print
+        summary: Print formatted type casted to the address or variable
+        args:
+          - name: type
+            type: RZ_CMD_ARG_TYPE_STRING
```

Comment:
I'm seeing this "type" arg more and more. I suggest you think about a new ARG_TYPE for these at some point during your refactoring. For example, RZ_CMD_ARG_TYPE_STRUCT_TYPE, RZ_CMD_ARG_TYPE_ENUM_TYPE, etc. This way, you can also have autocompletion easily once you implement the autocompletion function in cautocmplt.c for all commands that use the right type.

Reply:
Can autocompletion for e.g. `tl` (link type to the address or variable) combine both? For example, it can accept both structures and unions.

Reply:
Right now, no. You'd have to define another ARG_TYPE for that.
For example: `RZ_CMD_ARG_TYPE_STRUCT_TYPE`, `RZ_CMD_ARG_TYPE_ENUM_TYPE` and `RZ_CMD_ARG_TYPE_ANY_TYPE`

Reply:
What about this?

Reply:
I will do it in a separate PR